scripts/scripts.js

```diff
@@ -3,37 +3,37 @@ const speakers = [
     name: 'Chris Andreson',
     bio: 'Curator of TED',
     about: 'After a long career in journalism and publishing, Chris Anderson became the curator of the TED Conference in 2002 and has developed it as a platform for identifying and disseminating ideas worth spreading.',
-    picture: '/images/chris-anderson.jpg',
+    picture: '../images/chris-anderson.jpg',
```

Comment:
- [ ] IMPORTANT : these two dots are crushing the page, because of them the images are broken, because the route is wrong. When you put one dot it means one folder up, please delete one dot from every `img` url:

Debugged, with 1 dot  | With 2 dots
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/152653787-96e6bfa8-2dcc-4af8-a822-543d09aa8fc8.png) | ![image](https://user-images.githubusercontent.com/77038610/152653757-1f2274b2-9490-4d43-83db-89413066836f.png)

Please check the video where I am showing the issue of broken images and how to fix it : 

https://user-images.githubusercontent.com/77038610/152653902-e43fbcec-f8d7-4369-b5f3-ab906bc79423.mp4

You can also see that images in  about page are broken, because in css file the background image route is missing a dot: 

As it is: `background-image: url(/images/paste-event-2.jpg);`
How to fix: `background-image: url(./images/paste-event-2.jpg);`

Reply:
I´ve fixed the path of the images. Thanks.

Reply:
But... then (when I fdollowed the suggestions above regarding the images)I could not see the images anymore in the browser in my local repo... so I let it like: url(images/... )